diff --git a/kernel/selinux/sepolicy.h b/kernel/selinux/sepolicy.h
--- a/kernel/selinux/sepolicy.h
+++ b/kernel/selinux/sepolicy.h
@@ -5,6 +5,10 @@
 
 #include "ss/policydb.h"
 
+struct selinux_policy;
+struct selinux_policy *ksu_dup_sepolicy(struct selinux_policy *old_pol);
+void ksu_destroy_sepolicy(struct selinux_policy *pol);
+
 // Operation on types
 bool ksu_type(struct policydb *db, const char *name, const char *attr);
 bool ksu_attribute(struct policydb *db, const char *name);
